tfhe/src/integer/ciphertext/compressed_ciphertext_list.rs

```diff
@@ -176,6 +180,42 @@ impl CompressedCiphertextList {
             .map(|(blocks, kind)| T::from_expanded_blocks(blocks, kind))
             .transpose()
     }
+    #[cfg(feature = "gpu")]
+    pub fn get_decompression_size_on_gpu(
+        &self,
+        index: usize,
+        decomp_key: &CudaDecompressionKey,
+        streams: &CudaStreams,
+    ) -> Option<u64> {
+        self.get_blocks_of_size_on_gpu(index, decomp_key, streams)
+    }
+    #[allow(clippy::unnecessary_wraps)]
```

Comment:
I would agree with clippy that this function shoud return u64, not Option<u64>

Reply:
Actually in this function I do:
```
let preceding_infos = self.info.get(..index)?;
```
and I can't do that if I don't return an option.

Reply:
Well actually, what is the error clippy gives you here ? because the Option is needed here